examples/README.md

```diff
@@ -0,0 +1,93 @@
+items follow the recommendations for [Catalog Layout Best Practices](../best-practices.md#catalog-layout).
+
+## In Depth
+
+As mentioned above, the files in this examples directory form valid STAC implementations. They are all based on a 
+fictional remote sensing company called 'Remote Data', with a URL at remotedata.io. This domain has not been set up, so those links
+will not work, but any valid data provider should provide valid links to their homepage. 
+
+The examples use the `rd:` prefix to show how providers can use custom fields when there are not set fields. In the examples these
+do not link to a schema which is completely valid, but it is recommended that providers do write a JSON schema that can validate 
+their custom fields (we will work to add a schema in the future). 
+
+### Differences with STAC Best Practices
+
+One of the most important documents in this repository is the one about [best practices](../best-practices.md). It describes a number
+of practical recommendations gained by people actually implementing STAC. The core spec is designed to be as flexible as possible, so
+that it is not too rigid and unable to handle unanticipated needs. But we recommend following as many of the best practices as is 
+feasible, as it will help ensure various STAC tools work much better. The examples in this folder don't align with all the best
+practices, mostly because they are meant to demonstrate things as tersely as possible, and also because they live directly inside
+a github repository. As many people will look at these examples and take them as 'how things should be' we felt its important to
+highlight where things here differ from the actual best practices.
+
+#### Relative vs Absolute Links
+
+One of the [main recommendations](../best-practices.md#use-of-links) of the best practices document is to aim for consistency in 
+the use of relative versus absolute links. STAC implementations should use relative and absolute links as recommended by one of
+the 'catalog types': '[self-contained](../best-practices.md#self-contained-catalogs)', or '[published](../best-practices.md#published-catalogs)' - absolute or relative. 
+
+The catalogs contained here don't follow any of the recommendations in full. They are closest to a 'relative published catalog',
+as they use a self link at the root and then mostly have relative links. This enables them to be mostly portable, while also having
+a stable online location that can be referenced. But they use absolute links to their assets, due to the fact that they are stored in 
+github and we do not want to replicate the large asset files in every branch in github. A true relative published catalog will only
+have absolute links for the self, so that the rest is completely portable. 
+
+#### Catalog Layout
+
+Another important recommendations concerns the [layout of STAC catalogs](../best-practices.md#catalog-layout). This is important
```

Comment:
Why not just add a folder? It's just a single folder, right?

Reply:
I thought it was nice to come to a directory and see key examples right there. To be able to open them easily and compare, instead of having to dig through directories. I think with best practices we'd add a folder for each item, so it's more than a single folder.

Reply:
Indeed, it's a folder per item. So it depends on the outcome of #984? If we deploy and add the assets, then it would make sense to actually follow the best practices. If there's no assets, it doesn't make much sense to add a folder per item.

Nevertheless, I found the structure at the moment a bit confusing because there's no clear separation at first glance (GH schows you the files first and then you see the README)

Reply:
Yeah, I agree that if we deploy and add assets it would make sense to follow the best practices, at least in the deployed version. I think that tells a nice story - in github it's a self-contained catalog, for local use, and it's maybe a bit more ok that it's not quite best practices. But publishing would use pystac or another tool to format and publish, which would use the best practices, which is where it's more important to follow all the 'rules' - when you're exposing to the world.